src/barcodes/CODE93/constants.js

```diff
@@ -0,0 +1,79 @@
+// The position in the array is the (checksum) value
+export const SYMBOLS = [
+	'0', '1', '2', '3',
+	'4', '5', '6', '7',
+	'8', '9', 'A', 'B',
+	'C', 'D', 'E', 'F',
+	'G', 'H', 'I', 'J',
+	'K', 'L', 'M', 'N',
+	'O', 'P', 'Q', 'R',
+	'S', 'T', 'U', 'V',
+	'W', 'X', 'Y', 'Z',
+	'-', '.', ' ', '$',
+	'/', '+', '%',
+	// Only used for csum and multi-symbols character encodings
+	'($)', '(%)', '(/)', '(+)',
+	// Start/Stop
+	'\xff',
+	// Multi-sequence characters (Full ASCII Code 93)
+	'\x00', '\x01', '\x02', '\x03',
+	'\x04', '\x04', '\x06', '\x07',
+	'\x08', '\x09', '\x0a', '\x0b',
+	'\x0c', '\x0d', '\x0e', '\x0f',
+	'\x10', '\x11', '\x12', '\x13',
+	'\x14', '\x14', '\x16', '\x17',
+	'\x18', '\x19', '\x1a', '\x1b',
+	'\x1c', '\x1d', '\x1e', '\x1f',
+	'!', '"', '#', '&',
+	'\'', '(', ')', '*',
+	',', ':', ';', '<',
+	'=', '>', '?', '@',
+	'[', '\\', ']', '^',
+	'_', '`', 'a', 'b',
+	'c', 'd', 'e', 'f',
+	'g', 'h', 'i', 'j',
+	'k', 'l', 'm', 'n',
+	'o', 'p', 'q', 'r',
+	's', 't', 'u', 'v',
+	'w', 'x', 'y', 'z',
+	'{', '|', '}', '~',
+	'\x7f',
+];
+
+// Order matches SYMBOLS array
+export const BINARIES = [
+	'100010100', '101001000', '101000100', '101000010',
+	'100101000', '100100100', '100100010', '101010000',
+	'100010010', '100001010', '110101000', '110100100',
+	'110100010', '110010100', '110010010', '110001010',
+	'101101000', '101100100', '101100010', '100110100',
+	'100011010', '101011000', '101001100', '101000110',
+	'100101100', '100010110', '110110100', '110110010',
+	'110101100', '110100110', '110010110', '110011010',
+	'101101100', '101100110', '100110110', '100111010',
+	'100101110', '111010100', '111010010', '111001010',
+	'101101110', '101110110', '110101110', '100100110',
+	'111011010', '111010110', '100110010', '101011110',
+	['(%)', 'U'], ['($)', 'A'], ['($)', 'B'], ['($)', 'C'],
```

Comment:
I think it might be better to separate multi-symbols character to a separate structure. But I'm fine with this approach if you think it's better.